Extensions/find_inactives.js

```diff
@@ -290,7 +307,7 @@ XKit.extensions.find_inactives = new Object({
 
 		}
 
-		$(".find-inactives-blog").click(function() {
+		$(".find-inactives-blog-link").click(function() {
```

Comment:
why is this changed? doesn't this break clicking on the username and the days inactive?